trailblazer/store/crud/read.py

```diff
@@ -198,10 +195,11 @@ def get_job_by_id(self, job_id: int) -> Job | None:
             job_id=job_id,
         ).first()
 
-    def get_analyses_by_order_id(self, order_id: int) -> list[Analysis]:
+    def get_latest_analyses_in_an_order(self, order_id: int) -> list[Analysis]:
```

Comment:
```suggestion
    def get_latest_analyses_in_order(self, order_id: int) -> list[Analysis]:
```
the order is specified with id and not random, flavour

Reply:
Hmm yeah, true. Although `get_latest_analyses_in_order` makes it sound as if it is an ordered list rather than them belonging to the same order... @ChrOertlin

Reply:
How about "from_order"?